integration-tests/auth-with-rpc/src/pages/authenticated-query.tsx

```diff
@@ -17,16 +18,14 @@ function Content() {
       >
         logout
       </button>
-    </div>
+    </>
   )
 }
 
 function AuthenticatedQuery() {
   return (
     <div id="page">
-      <Suspense fallback={"Loading..."}>
-        <Content />
-      </Suspense>
+      <Content />
```

Comment:
When the Suspense wrapper is gone, those two methods can be collapsed/made one, right?

Reply:
yeah we could, I just wanted to make as minimal changes to the integration tests to also test the `useQuery`.